src/EnergyPlus/DXCoils.cc

```diff
@@ -12339,7 +12339,9 @@ namespace DXCoils {
                 HeatReclaimDXCoil(DXCoilNum).AvailCapacity = DXCoil(DXCoilNum).TotalCoolingEnergyRate + DXCoil(DXCoilNum).ElecCoolingPower;
 
                 // Waste heat calculation
-                if (DXCoil(DXCoilNum).MSHPHeatRecActive) {
+                // TODO: waste heat not considered even if defined in Cooling:DX:MultiSpeed, N16, \field Speed 1 Rated Waste Heat Fraction of Power
+                // Input
+                if (DXCoil(DXCoilNum).FuelType != FuelTypeElectricity) {
```

Comment:
@mjwitte I was hoping you can take a look at this piece. I checked you previous comments on considering waste heat for Cooling:DX:MultiSpeed in non- FuelTypeElectricity mode. But I found this part is currently not calculated even when user defines, as the calculation is only triggered when MSHPHeatRecActive.

Would you mind taking a look at whether my understanding and modifications are in the right direction? Thanks very much.